upload-action/package.json

```diff
@@ -28,6 +28,7 @@
     "typescript": "^5.9.3"
   },
   "scripts": {
+    "postinstall": "cd ../ && npm run build",
```

Comment:
this is being replaced with https://github.com/filecoin-project/filecoin-pin/pull/170